src/site/static/eligibility.liquid

```diff
@@ -1871,8 +1871,29 @@ pageClass: "page-public-assistance"
   }
 
   function gaEligible() {
-    // TODO
-    return false;
+    // https://socialservices.sccgov.org/financial-assistance/financial-assistance-adults-without-dependent-children-general-assistance
+    const age = document.getElementById("age").value || null;
+    const numDependents = document.querySelectorAll('input[id^="hh-member-dependent"]:checked').length;
+  
+    const MIN_GA_ELIGIBLE_AGE = 18;  // Years
+    const MAX_GA_ELIGIBLE_AGE = 64;  // Years
+    const NUM_OF_DEPENDENTS = 0;     // None
+    const MAX_RESOURCES = 500;       // USD Combined household assets
+    const MAX_GROSS_INCOME = 150;    // USD per month
+
+    return (
+      and(
+        ge(age, MIN_GA_ELIGIBLE_AGE),
+        le(age, MAX_GA_ELIGIBLE_AGE),
+        eq(numDependents, NUM_OF_DEPENDENTS),
+        le(totalResources(), MAX_RESOURCES),
+        le(grossIncome(), MAX_GROSS_INCOME),
+        or(
+          !document.getElementById("not-citizen").checked,
+          document.getElementById("permanent_resident").checked,
```

Comment:
Eventually we may need to add more detail to the immigration status form page, but for now, I think you can also add `qualified_noncitizen_gt5y` and `qualified_noncitizen_le5y`.  The GA immigration eligibility requirements say "qualified citizen" and I still need to work out what that means exactly.